dev/com.ibm.ws.security.wim.adapter.ldap_fat/bnd.bnd

```diff
@@ -23,7 +23,7 @@ tested.features: \
     vmmapi-1.0
 
 -buildpath: \
-    com.unboundid:unboundid-ldapsdk;version=latest,\
+    com.unboundid:unboundid-ldapsdk;version=4.0.14,\
```

Comment:
We are changing these all over, but I wonder if we wouldn't be better suited to just include the JAR in the com.ibm.ws.com.unboundid artifact JAR, then we wouldn't ever need to update these again when we need to update to a newer version.

You can use the "Include-Resource:" directive in the BND file and it will include the resource in the generated JAR. For example:

```
Include-Resource: \
    @${com.unboundid:unboundid-ldapsdk;4.0.14;EXACT}
```